code/lib/preview-web/src/PreviewWeb.test.ts

```diff
@@ -1,5 +1,5 @@
 /// <reference types="@types/jest" />;
-
+import { jest, jest as mockJest, it, describe, beforeEach, afterEach, expect } from '@jest/globals';
```

Comment:
do we need this?

Reply:
Strictly no, but I prefer it, rather than having global symbols available.